packages/istanbul-api/lib/reporter.js

```diff
@@ -6,7 +6,8 @@ var path = require('path'),
     configuration = require('./config'),
     inputError = require('./input-error'),
     libReport = require('istanbul-lib-report'),
-    libReports = require('istanbul-reports');
+    libReports = require('istanbul-reports'),
+    minimatch = require('minimatch');
```

Comment:
`minimatch` needs to be added to packages/istanbul-api/package.json as a `dependency`.

Reply:
Done!